src/components/Savings/SavingsCard/SavingsCardRow/index.tsx

```diff
@@ -70,12 +79,9 @@ export const SavingsCardRow = ({
                             width="130px"
                             size="sm"
                             borderRadius="6px"
-                            requireChain={'CELO'}
+                            requireChain={SupportedV2Networks[requiredChain] as keyof typeof SupportedV2Networks}
```

Comment:
i saw this in many places maybe declare custom type ?

```typescript
export type SupportedChain = keyof typeof SupportedV2Networks;

// ...
const requireChain = SupportedV2Networks[requiredChain] as SupportedChain
// ...
requireChain={requireChain}

Reply:
ye, moved this into modalbutton now